Makefile

```diff
@@ -87,25 +87,50 @@ ALL_FILES_WIN=\
     batavia\core\Frame.js \
     batavia\core\Generator.js \
     batavia\core\PYCFile.js \
-    batavia\VirtualMachine.js \
+    batavia\VirtualMachine.js
 
-.PHONY: all clean
+EXTRA_FILES=\
+		batavia/modules/stdlib/colorsys.js \
+		batavia/modules/stdlib/copyreg.js \
+		batavia/modules/stdlib/this.js
 
-all: batavia.js batavia.min.js
+EXTRA_FILES_WIN=\
+		batavia\modules\stdlib\colorsys.js \
+		batavia\modules\stdlib\copyreg.js \
+		batavia\modules\stdlib\this.js
+
+
+all: batavia-all.js batavia-all.min.js batavia.js batavia.min.js
+
+.PHONY: all clean stdlib
+
+stdlib:
+	python compile_stdlib.py
```

Comment:
is it the intent to have both a make point for this, and also distribute the compiled files out of this function? (You've also committed the stdlib artefacts this step creates)